style.css

```diff
@@ -0,0 +1,635 @@
+  text-align: center;
+}
+
+.p-contact {
+  font-size: 12px;
+  padding-bottom: 20px;
+}
+
+.p-contact a {
+  font-weight: 800;
+  color: #272a31;
+}
+
+.mp-title {
+  margin: 20px 20% 0 20%;
+  font-weight: 400;
+  font-size: 24px;
+  color: #272a31;
+}
+
+.middle-section {
+  border-bottom: 1px solid darkgray;
+}
+
+.prev-look {
+  font-size: 12px;
+  padding: 0 7%;
+}
+
+.summit-img {
+  background-color: #972a16;
+  background-image:
+    linear-gradient(
+      to right,
+      rgba(151, 42, 22, 0.6),
+      rgba(151, 42, 22, 0.6)
+    ),
+    url(/images/firstBackground.svg);
+  background-repeat: no-repeat;
+
+  /* background-size: ; */
+
+  /* height: 30vh; */
+
+  /* width: 86%; */
+
+  /* margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white; */
+}
+
+.summit-img1 {
+  background-image:
+    linear-gradient(
+      to right,
+      rgba(151, 42, 22, 0.6),
+      rgba(151, 42, 22, 0.6)
+    ),
+    url(./images/firstBackground.svg);
+}
+
+.summit-img h3 {
+  font-size: 30px;
+  margin-bottom: 20px;
+}
+
+.summit-img p {
+  font-size: 13px;
+}
+
+.blur {
+  margin-bottom: 25px;
+}
+
+.partners {
+  background-color: #414246;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  padding: 3% 3%;
+}
+
+.partners .partners-logo {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 20px;
+  justify-content: center;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 125px;
+  padding: 4% 4%;
+}
+
+.menu {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(151, 42, 22, 0.6);
+  backdrop-filter: blur(7px);
+  height: 100vh;
+  animation: growDown 300ms ease-in-out forwards;
+  transform-origin: top center;
+}
+
+@keyframes growDown {
+  0% {
+    transform: rotateX(-90deg);
+  }
+
+  70% {
+    transform: rotateX(20deg);
+  }
+
+  100% {
+    transform: rotateX(0deg);
+  }
+}
+
+.menu .xicon {
+  height: 20px;
+  width: 20px;
+  margin-left: 86%;
+  margin-top: 7.5%;
+}
+
+.menu.visible {
+  display: block;
+  z-index: 20;
+}
+
+.menu .menu-li {
+  list-style-type: none;
+  margin-top: 13%;
+}
+
+.menu .menu-li a {
+  text-decoration: none;
+  font-size: 25px;
+  display: flex;
+  justify-content: center;
+}
+
+.menu .menu-li a:hover {
+  font-weight: bolder;
+}
+
+.menu .nav-link {
+  color: #fff;
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.footer-rights {
+  font-size: 12px;
+}
+
+.speakers {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.speakers .mp-title {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.speakers .spk-hr {
+  width: 48%;
+  margin: 6px auto 54px auto;
+}
+
```

Comment:
Please style your speaker section to look almost like the of the [Figmal Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
### **Figmal Design**
![figSpeaker](https://user-images.githubusercontent.com/55596970/181653512-798560cb-c6b4-4541-8ff5-99286da6c758.png)

### **Your Design**
![YoSpeaker](https://user-images.githubusercontent.com/55596970/181653562-b76da2c6-3d92-4f18-90ef-5619138682d3.png)